src/Shortcode/Twig.php

```diff
@@ -179,6 +179,7 @@ private function addGlobals( \Twig_Environment $twigEnv ) {
             'path_and_query' => $request->getRequestUri(),
             'query' => $request->getQueryString()? '?' . $request->getQueryString() : '',
             'url' => $request->getUri(),
+            'referer' => $request->headers->get('referer'),
```

Comment:
Please observer the code style used throughout the project.

```suggestion
            'referer' => $request->headers->get( 'referer' ),
```